wpeverest/user-registration
# Enhance - Extension page design and addons installation.

### All Submissions:






### Changes proposed in this Pull Request:
Previously users do not have options to download, install and activate our addons and have to manually download them from their account page from wpeverest.com. This PR provides users with the option to download, Install, activate and deactivate our addons from our extensions page menu.

### How to test the changes in this Pull Request:

1. First of activate license of User registration otherwise you will see only the upgrade plan button on each addon.
2. Visit the addons page and click on the Refresh Addons button and verify if all the images, contents, and preview links of addons are intact and working as expected.
3. Verify if the design of the page has changed or not.
4. Verify if you are able to Install, activate and deactivate addons properly that are in your current plan.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Extension page design and addons installation.
